File: src/net/novauniverse/games/survivalgames/event/SurvivalgamesCountdownEvent.java
package net.novauniverse.games.survivalgames.event;

import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;

public class SurvivalgamesCountdownEvent extends Event {
	private static final HandlerList HANDLERS_LIST = new HandlerList();

	private int timeLeft;

	public SurvivalgamesCountdownEvent(int timeLeft) {
		this.timeLeft = timeLeft;
	}

	public int getTimeLeft() {
		return timeLeft;
	}

	@Override
	public HandlerList getHandlers() {
		return HANDLERS_LIST;
	}

	public static HandlerList getHandlerList() {
		return HANDLERS_LIST;
	}
}

File: src/main/kotlin/net/novauniverse/games/survivalgames/event/SurvivalgamesCountdownEvent.kt
package net.novauniverse.games.survivalgames.event

import org.bukkit.event.Event
import org.bukkit.event.HandlerList

class SurvivalgamesCountdownEvent(val timeLeft: Int): Event() {
    override fun getHandlers(): HandlerList {
        return HANDLERS
    }

    companion object {
        private val HANDLERS = HandlerList()

        @JvmStatic
        fun getHandlerList(): HandlerList {
            return HANDLERS
        }
    }
}